docs/fn-2.0/README.md

```diff
@@ -329,58 +329,70 @@ Picks runner to run it on and passes along event
 
 ## CLI changes
 
-### fn deploy
+### NEW IN 2.0
 
-func.yaml barely changes:
+The main changes here are related to [splitting functions and routes](https://github.com/fnproject/fn/issues/817).
+A route is now called a trigger and the route/trigger definition is no longer part of the function definition as it was before.
+
+### Function Definition File
+
+`func.yaml` barely changes, just removes route/trigger specific things. 
+
+A function does not know about triggers so it only has the information required to run the function.
 
 ```yaml
 function:
   name: yodawg
-  image: hub.docker.com/fnproject/hello
```

Comment:
i think we should separate `name` and `image` still here, as it is in the API. aside from the addressability issue of e.g. `/ns/1/funcs/quay.io/rdallman/yodawg` -- i am still salty about the registry business https://github.com/fnproject/cli/pull/144 and for a cloud implementer it seems likely that many users will use a non-dockerhub registry, and we should try to be more friendly in that direction.